bin/docpub.js

```diff
@@ -10,18 +10,38 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
+    .option('--config-path <configPath>', 'path for the config')
+    .option('-v --verbose', 'verbose output');
+
+program.on('--help', () => {
+    console.log('  Overriding config');
+    console.log('    To override any config option use full option path converted to --kebab-case');
+    console.log('');
+
+    console.log('    Examples:');
+    console.log('      docpub --username user@example.com');
+    console.log('      docpub --token abcdefghijklmn12345');
+    console.log('');
+    console.log('    You can also use environment variables converted to snake_case with');
```

Comment:
@ADHER that's true, but it's better to add features when they needed. So still suggest to leave now as-is.